_guides/analytics/analytics_basics.md

```diff
@@ -0,0 +1,229 @@
+Page view tracking works in `amp-analytics` too.
+But you can also track user engagement with any type of page content,
+including clicks on links, buttons, videos.
+And you can measure how far on the page the user scrolled,
+whether or not the user engaged with social media, and more.
+[Deep Dive into Amp Analytics](/docs/guides/analytics/deep_dive_analytics.html).
+
+As part of integrating with the AMP platform,
+many analytics providers have simplified `amp-analytics` configurations
+so that's it's easy to capture data and push to their tracking tools.
+Access vendor documentation from the
+[amp-analytics specification](/docs/reference/extended/amp-analytics.html).
+
+**Note:** You can use both `amp-pixel` and `amp-analytics` in your pages:
+`amp-pixel` for simple page view tracking,
+and `amp-analytics` for everything else.
+Keep in mind that simpler AMP pages are better for users,
+so if you don’t need the extra tags, don’t use them.
+
+## Create a simple analytics configuration
+
+Learn how to create a simple
+[amp-pixel](/docs/reference/amp-pixel.html) and
+[amp-analytics](/docs/reference/extended/amp-analytics.html) configuration.
+
+### Simple amp-pixel configuration
+
+To create a simple `amp-pixel` configuration,
+insert something like the following into the body of your AMP page:
+
+{% highlight html linenos %}
+<amp-pixel src="https://foo.com/pixel?RANDOM"></amp-pixel>
+{% endhighlight %}
+
+In this example,
+the page view data gets sent to the defined URL, along with a random number.
+The `RANDOM` variable is one of many
+[supported variables in the AMP platform](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md).
+Learn more about
+[Variable substitution](/docs/guides/analytics/analytics_basics.html#variable-substitution) here.
+
+The [amp-pixel](/docs/reference/amp-pixel.html)
+component is built-in,
+so you won’t need an inclusion declaration.
+But you should place the `amp-pixel` tag as close as possible
+to the start of your body.
+The tracking pixel will only fire when the tag comes into view itself.
+If `amp-pixel` is positioned near the bottom of the page,
+it may not fire.
+
+### Simple amp-analytics configuration
+
+To create a simple
+[amp-analytics](/docs/reference/extended/amp-analytics.html) configuration,
+you must first include this `custom-element` declaration
+in the head of the AMP document:
+
+{% highlight html linenos %}
+<script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+{% endhighlight %}
+
+Similar to the [amp-pixel example](/docs/guides/analytics/analytics_basics.html#simple-amp-pixel-configuration),
+the following `amp-analytics` sample tracks page views and
+sends the data to a defined URL along with a random ID: 
+
+{% highlight html linenos %}
+<amp-analytics>
+<script type="application/json">
+{
+  "requests": {
+    "pageview": "https://foo.com/pixel?RANDOM",
+  },
+  "triggers": {
+    "pixel emulation": {
```

Comment:
Alternatively, you could call it "fire on a page view" (or something similarly verbose) to distinguish from the "pageview" trigger keyword.